web/client/translations/data.fr-FR.json

```
@@ -1679,7 +1679,16 @@
                 "syncOnMap": "Synchroniser la carte avec un filtre",
                 "enableTimeSync": "Synchroniser avec l'heure ou l'intervalle de temps sélectionné",
                 "disableTimeSync": "Ne pas synchroniser avec l'heure ou l'intervalle sélectionné",
-                "createNewChart": "Créer un graphique pour le calque sélectionné"
+                "createNewChart": "Créer un graphique pour le calque sélectionné",
+                "enableSnapping": "Activer l'accrochage",
+                "disableSnapping": "Désactiver la capture",
```

- [ ] these doesn't seem constistent.  Trying to guess :smile: 
```suggestion
                "enableSnapping": "Activer l'accrochage",
                "disableSnapping": "Désactiver l'accrochage",
```